app/constants/languages.js

```diff
@@ -77,6 +75,7 @@ const languages = [
   { value: 'moh-ca', label: 'Mohawk (Mohawk)' },
   { value: 'mn', label: 'Mongolian' },
   { value: 'ne-np', label: 'Nepali (Nepal)' },
+  { value: 'nso', label: 'Northern Sotho / Sesotho sa Leboa' },
```

Comment:
Can we please not use colonial names?

Reply:
As I wrote in the PR text, I'm opting to include _both_ the official ISO Language Name and the local/native name. Including both names is useful -- it resolves possible confusion in the case of "Sesotho" -- Sesotho = Southern Sotho = `st` vs. Sesotho sa Leboa = Northern Sotho = `nso`.

Reply:
The ISO name is not official. Can we use the names that we’ve been asked to use by people who speak these languages.

Reply:
Also, ‘native name’????